pkg/config/node/minimum_kubelet_version.go

```diff
@@ -0,0 +1,56 @@
+package node
+
+import (
+	"context"
+	"fmt"
+	"strings"
+
+	"github.com/blang/semver/v4"
+
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/util/validation/field"
+	corev1client "k8s.io/client-go/kubernetes/typed/core/v1"
+)
+
+func ValidateMinimumKubeletVersion(nodesGetter corev1client.NodesGetter, minimumKubeletVersion string) *field.Error {
+	// unset, no error
+	if minimumKubeletVersion == "" {
+		return nil
+	}
+
+	fieldPath := field.NewPath("spec", "minimumKubeletVersion")
+	nodes, err := nodesGetter.Nodes().List(context.TODO(), metav1.ListOptions{})
```

Comment:
Can this be made to accept the lister interface instead of the client interface? Listing all nodes as part of validating an API field is probably more expensive than we would like.

Reply:
updated!